### ISSUE TITLE
Latching ability for Publishers

### ISSUE THREAD

**Issue by Voidminded** [ros-drivers/rosserial#372]
Hi,

I think It would be very helpful to add latching ability to the Publishers. As the serial communication should be kept minimal, being able to latch a message could reduce the serial communication a lot for topics that should be checked frequently but being modified rarely. Especially in systems containing nodes which keep connecting and disconnecting to the system.

**Comment by EliosSs** [comment]
I agree with this. It is not possible yet to latch topics from rosserial or rosserial arduino?

**Comment by SubaruArai** [comment]
I guess this needs a protocol update, and since there will be no new official ROS1 version, while it is technically possible, it won't be practically possible.
Although I would like to have a latching capability too.
